apps/demos/Demos/DataGrid/CustomEditors/ReactJs/EmployeeDropDownBoxComponent.js

```diff
@@ -7,7 +7,8 @@ import DropDownBox from 'devextreme-react/drop-down-box';
 const dropDownOptions = { width: 500 };
 const ownerLabel = { 'aria-label': 'Owner' };
 const EmployeeDropDownBoxComponent = (props) => {
-  const [selectedRowKeys, setSelectedRowKeys] = useState([props.data.value]);
+  const [selectedRowKeys, setSelectedRowKeys] = useState(() =>
```

Comment:
Let's move on the initial state value to a separate variable:

```suggestion
  const { data: { value: dataValue } } = props;
  const initialSelectedRowKeys = dataValue !== null && dataValue !== undefined ? [dataValue] : []
  const [selectedRowKeys, setSelectedRowKeys] = useState(initialSelectedRowKeys);
```